content/doc/book/installing/index.adoc

```diff
@@ -238,6 +238,52 @@ If your `/etc/init.d/jenkins` file fails to start Jenkins, edit the `/etc/defaul
 Here, "8081" was chosen but you can put another port available.
 ====
 
+===== Installation of Java
+
+Jenkins requires Java in order to run, yet certain distributions don't include this by default and  link:doc/administration/requirements/java/[some Java versions are incompatible] with Jenkins.
+
+There are multiple Java implementations which you can use. link:https://openjdk.java.net/[OpenJDK] is the most popular one at the moment, we will use it in this guide.
+
+To install the Open Java Development Kit (OpenJDK) run the following:
+
+* Update the repositories 
+[source,bash]
+----
+sudo apt update
+----
+* search of all available packages:
+[source,bash]
+----
+sudo apt search openjdk
+----
+* Pick one option and install it: 
+[source,bash]
+----
+sudo apt install openjdk-8-jdk
+----
+* Confirm installation:
+[source,bash]
+----
+sudo apt install openjdk-8-jdk 
+----
+* checking installation:
+[source,bash]
+----
+java -version
+----
+* the result must be something like:
+[source,bash]
+----
+openjdk version "1.8.0_252"
+OpenJDK Runtime Environment (build 1.8.0_252-8u252-b09-1ubuntu1-b09)
+OpenJDK 64-Bit Server VM (build 25.252-b09, mixed mode)
+----
+
+[NOTE]
+====
+Why to use `apt` command and not `apt-get` or other command? Since 2014 the apt command it is avaliable to use. They both have a similar command structure but apt was created to be more pleasant and complete option for most of common users so there is no need to know tons of command option to do simple tasks like install, search or remove softwares from your operation system.
```

Comment:
Phrasing changes and make it easier to review by placing a single sentence per line.
```suggestion
Why use `apt` and not `apt-get` or another command?
The apt command has been available since 2014.
It has a command structure that is similar to `apt-get` but was created to be a more pleasant experience for typical users.
Simple software management tasks like install, search and remove are easier with `apt`.
```